lib/pdf_fill/forms/va686c674.rb

```diff
@@ -1545,6 +1589,10 @@ def merge_previous_marriage_helpers
             # 'annulment' => select_radio_button(reason_marriage_ended == 'ANNULMENT'),
             'other' => select_radio_button(reason_marriage_ended == 'Other')
           }
+
+          # extract country: FE uses 3 char country codes, but pdf expects 2 char country code
+          spouse['start_location']['country'] = extract_country(spouse['start_location'])
+          spouse['end_location']['country'] = extract_country(spouse['end_location'])
```

Comment:
Could we (should we) add spec coverage for the 3-char => 2-char conversion behavior here? Same for the other additions of `extract_country` in this file.

Reply:
Same comment as above.  The tests are covered in filler_spec when the PDFs get generated based off the data in the payload.  Please see screenshots for examples of how the 3 digit country code is being extracted (and tested) as 2 digit country codes in the PDF:

<img width="868" alt="PreviousMarriageStartLocation" src="https://user-images.githubusercontent.com/53828142/89941444-e1217180-dbcf-11ea-9812-7233f1277f9f.png">


<img width="870" alt="PreviousMarriageEndLocation" src="https://user-images.githubusercontent.com/53828142/89941453-e54d8f00-dbcf-11ea-8f57-fd09e117f485.png">